client/me/help/chat-closure-notice/index.jsx

```diff
@@ -0,0 +1,74 @@
+/**
+ * Fixed notice about upcoming support closures
+ */
+
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import i18n, { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import FormSectionHeading from 'components/forms/form-section-heading';
+import olarkStore from 'lib/olark-store';
+import { title, upcoming, closed } from './messages';
+
+const Notice = localize( ( { translate, closedFrom, closedTo, reason } ) =>
+	<div className="chat-closure-notice">
+		<FormSectionHeading>{ title( { translate, closedFrom, closedTo, reason } ) }</FormSectionHeading>
+		<p>
+			{
+				i18n.moment().isBefore( closedFrom )
```

Comment:
I think this would be useful, but that's something we can always implement when we need it. Usually the closures are decided only a few weeks in advance, so it's not a problem just yet.